content/applications/websites/website/configuration/translate.rst

```diff
@@ -2,111 +2,138 @@
 Translations
 ============
 
-Your website is displayed in the language that matches your visitor’s browser. If the browser’s
-language has not been installed and added to your website, the content is shown in the
-:ref:`default language <translate/default-language>`. When additional languages are installed, users
-can choose their preferred language using the :ref:`language selector <translate/language-selector>`.
+The website can be translated into multiple languages to allow visitors to view the content in their
+preferred language.
 
-The :ref:`Translate <translate/translate>` feature on your website allows automatic translation of
-standard terms and provides a tool for manual content translation.
+
+To make the content available in other languages, install the required languages and add them to the
+website. Once this is done, a :ref:`Language selector <translate/language-selector>` is
+automatically added to the website, allowing visitors to choose their preferred language.
 
 Install languages
 =================
```

Comment:
```suggestion
Configuration
=============
```
I would simply use "configuration" here, making it clear it's the required configuration step before translating.